staging/src/k8s.io/client-go/examples/in-cluster-client-configuration/README.md

```diff
@@ -37,7 +37,7 @@ kubectl create clusterrolebinding default-view --clusterrole=view --serviceaccou
 
 Then, run the image in a Pod with a single instance Deployment:
 
-    kubectl run --rm -i demo --image=in-cluster
+    kubectl run --rm -i demo --image=in-cluster:latest --image-pull-policy=IfNotPresent
```

Comment:
i think if tag is not provided, the default is 'latest', do you need to give it explicitly?

why do you need to provide --image-pull-policy=IfNotPresent?

Reply:
I think  'latest' is not necessary.

I run this example in minikube and meet some problem.

After i  build image as follow which this file memtioned before.
```
 eval $(minikube docker-env)
 docker build -t in-cluster .	    docker build -t in-cluster .
```
when i run this command `kubectl run --rm -i demo --image=in-cluster`,   the pod status is  ImagePullBackOff.  

After I add `--image-pull-policy=IfNotPresent`  ,the pod status is running.

ref  https://stackoverflow.com/a/49478889/4910370

Reply:
change the `docker build` as well in line 25 to
```
    docker build -t in-cluster .
```
And this line should be like 
`kubectl run --rm -i demo --image=in-cluster:dev`
    
It will use `--image-pull-policy=IfNotPresent` by default.
But for image without tag as well as `latest` tag, the image pull policy is by default Always.

https://kubernetes.io/docs/concepts/containers/images/#updating-images

Reply:
i have change the line